mizar/daemon/interface_service.py

```diff
@@ -179,6 +228,16 @@ def _ConfigureTransitAgent(self, interface):
         self.rpc.update_agent_metadata(interface)
         self.rpc.update_ep(interface)
 
+    def _DeleteTransitAgent(self, interface):
+        self.rpc.load_transit_agent_xdp(interface)
+
+        for bouncer in interface.bouncers:
+            self.rpc.delete_agent_substrate_ep(
+                interface.veth.peer, bouncer.ip_address)
```

Comment:
Well, this can only happen from the operator. Because we need to maintain a reference count of the usage of this substrate endpoint!